### DISCUSSION TITLE
Azure DevOps Server 2022.0.1 Migration Error

### DISCUSSION THREAD

**Discussion by ancharbo** [github/gh-actions-importer#252] [Feedback]
Trying to do a migration from ADO Server, no issues while configure or audit, but receiving an error when trying to run a dry-run or migrate:

```
There was an error extracting the Azure DevOps pipeline (https://tfs.xxxxxxxxx.corp/tfs/CS/ALM/_build?definitionId=22)
Message: There was an error retrieving file contents from Azure DevOps (https://tfs.xxxxxxxxx.corp/tfs/CS/ALM/_git/Mobile.App.UI)
Message: Unable to parse Azure DevOps project URL. Ensure it is in the format of https://dev.azure.com/:organization/:project_name/git/:repository_name"
```

The importer is able to determine which repo the pipelineID resides within, so I don't think it is a PAT scoping issue. Is this a limitation of the migrator or is something misconfigured?  I suspect it has something to do with the on-prem URL. Thanks

**Comment by Chaseshak** [comment]
Hi! Can you please make a ticket through [GitHub support](https://support.github.com/request/actions)?

Thanks!
